src/System.Management.Automation/engine/runtime/Operations/MiscOps.cs

```diff
@@ -403,31 +403,78 @@ private static string GetParameterText(string parameterName)
 
                 CommandProcessorBase commandProcessor = null;
                 CommandRedirection[] commandRedirection = null;
-                for (int i = 0; i < pipeElements.Length; i++)
+
+                if (((PipelineAst)pipeElementAsts[0].Parent).BackgroundProcess)
                 {
-                    commandRedirection = commandRedirections != null ? commandRedirections[i] : null;
-                    commandProcessor = AddCommand(pipelineProcessor, pipeElements[i], pipeElementAsts[i],
-                                                  commandRedirection, context);
+                    // For background jobs rewrite the pipeline as a Start-Job command
+                    var scriptblockBodyString = ((PipelineAst)pipeElementAsts[0].Parent).Extent.Text;
+                    Token[] tokens;
+                    ParseError[] errors;
+                    var pipelineBodyAst = Parser.ParseInput(scriptblockBodyString, out tokens, out errors);
+                    // Prefix variables in the scriptblock with $using: 
+                    var variables = pipelineBodyAst.FindAll(x => x is VariableExpressionAst, true).Reverse();
+                    foreach (var v in variables)
+                    {
+                        var vName = v.Extent.Text.Substring(1);
+                        // Skip variables that don't exist
+                        if (funcContext._executionContext.EngineSessionState.GetVariable(vName) == null)
+                            continue;
+                        // Skip PowerShell magic variables
+                        if (Regex.Match(vName,"^PID$|^PS|^HOST$", RegexOptions.IgnoreCase|RegexOptions.CultureInvariant).Success == false)
+                        {
+                            scriptblockBodyString = scriptblockBodyString.Substring(0, v.Extent.StartOffset) +
+                                "$using:" + vName + scriptblockBodyString.Substring(v.Extent.EndOffset);
```

Comment:
This is pretty inefficient in that it creates a bunch of strings.

It would be better to start with a copy of the script block body in a `StringBuilder` and insert `using` as appropriate.